packages/synapse-react-client/src/synapse-queries/entity/useGetQueryResultBundle.ts

```diff
@@ -50,11 +50,14 @@ export default function useGetQueryResultBundle(
   )
 }
 
-function _useGetQueryResultBundleWithAsyncStatus(
+function _useGetQueryResultBundleWithAsyncStatus<
+  TData = AsynchronousJobStatus<QueryBundleRequest, QueryResultBundle>,
+>(
```

Comment:
Needed to add generics to use the `select` option in `useGetActionsRequiredForTableQuery`